### PR TITLE
Bug ie edge cases

### PR DESCRIPTION
**Pull request by grahamhency**
## Problem
Many IE bugs

## Solution
Squashed them all

### CODE REVIEW

File: projects/go-tester/browserslist [hunk 1]
```diff
@@ -8,4 +8,4 @@
 last 2 versions
 Firefox ESR
 not dead
-not IE 9-11
\ No newline at end of file
+IE 9-11
```
Comment (AlexOverbeck) [inline projects/go-tester/browserslist]: Do we need to specify `9-11` here or can we just do `11`?

File: projects/go-tester/src/polyfills.ts [hunk 2]
```diff
@@ -19,20 +19,20 @@
...
+import 'core-js/es6/parse-float';
+import 'core-js/es6/number';
+import 'core-js/es6/math';
+import 'core-js/es6/string';
+import 'core-js/es6/date';
+import 'core-js/es6/array';
+import 'core-js/es6/regexp';
+import 'core-js/es6/map';
+import 'core-js/es6/weak-map';
+import 'core-js/es6/set';
```
Comment (AlexOverbeck) [inline projects/go-tester/src/polyfills.ts]: This is kind of a bummer.

File: package-lock.json [hunk 3]
```diff
@@ -1440,7 +1440,6 @@
       "resolved": "https://registry.npmjs.org/are-we-there-yet/-/are-we-there-yet-1.1.5.tgz",
```
Comment (AlexOverbeck) [inline package-lock.json]: Looks like you maybe used the wrong version of npm to install these modules. You shouldn't have this big of a diff.

File: projects/go-tester/browserslist (hunk 1 above)
Comment (grahamhency) [reply projects/go-tester/browserslist]: Not sure, I was just doing this for testing purposes and would be dependent upon the implementor to do. I bet we can just do `IE 11`

**Pull request merged**